tests/xfstests/run.pm

```diff
@@ -277,11 +277,13 @@ sub run {
 
         sleep(1);
         select_console('root-console');
-        # Save kdump data to KDUMP_DIR
-        unless (save_kdump($test, $KDUMP_DIR)) {
-            # If no kdump data found, write warning to log
-            my $msg = "Warning: $test crashed SUT but has no kdump data";
-            script_run("echo '$msg' >> $LOG_DIR/$category/$num");
+        # Save kdump data to KDUMP_DIR if not set "NO_KDUMP"
+        unless (get_var("NO_KDUMP")) {
```

Comment:
Like you said kdump bug block the test on aarch64, so no need create a new var to check it, just use `check_var('ARCH', 'aarch64')` to do the condition check. Also does it report a bug? if so, better to record soft fail for aarch64.

Reply:
To record soft fail will not invole in testing filesystem, it can be add into test kdump. 
Here only see if it's aarch64, then open the switch. I think keep this var is better, if other frame has similar issue I can just modify one place to control all kdump in my test. I'll use a check_var('ARCH', 'aarch64') to set_var('NO_KDUMP', 1) to control all this behavior in front of xfstests.